### PR TITLE
fix(sql): compound_statement -> block

### PR DESCRIPTION
**Pull request by dmfay**
a little more reorganizing; we've copied the indents file into tree-sitter-sql and it tests okay there.

### CODE REVIEW

**Comment by clason** [comment]
@matthias-Q

File: queries/sql/indents.scm [hunk 1]
```diff
@@ -7,8 +7,7 @@
  (insert)
 ] @indent.begin
 
-
-(compound_statement
+(block
```
Comment (lucario387) [inline queries/sql/indents.scm:10]: Hmmm, which one is more narrowed down. Is it `compound_statement` or `block`

as in, is the s-expr tree `(compound_statement (block (keyword_begin)))` or `(block (compound_statement (keyword_begin)))`

Comment (lucario387) [reply queries/sql/indents.scm:10]: Ahhh, I see from the commit logs of tree-sitter-sql now, `compound_statement` is now renamed and refactored into `block`, is it correct? @dmfay

Comment (clason) [reply queries/sql/indents.scm:10]: I think this was just a rename in the grammar?

Comment (dmfay) [reply queries/sql/indents.scm:10]: that's correct, `block` is a more common name for the expression formerly called `compound_statement`. Only the name has changed, nothing's more specific or more general.

**Review by clason — APPROVED** [review]

**Review by lucario387 — APPROVED** [review]

**Pull request merged**